pkgs/applications/version-management/bitkeeper/default.nix

```diff
@@ -0,0 +1,48 @@
+{stdenv, fetchurl, perl, gperf, bison, groff,
+ pkgconfig, libXft, fontconfig, pcre, libtomcrypt, libtommath, lz4, zlib}:
+
+stdenv.mkDerivation rec {
+  name = "bitkeeper-7.3ce";
+  base = "bk-7.3ce";
+  enableParallelBuilding = true;
+  src = fetchurl {
+    url = "https://www.bitkeeper.org/downloads/7.3ce/${base}.tar.gz";
+    sha256 = "13249636f4b5b39f1d64b9f6bf682ee2dce53db17cdd8aa4cd9019e65252cabb";
+  };
+  sourceRoot = "${base}/src";
+
+  buildInputs = [ perl gperf bison groff libXft pkgconfig
+	      pcre libtomcrypt libtommath lz4];
+
+  postPatch = ''
+	substituteInPlace port/unix_platform.sh \
+		--replace /bin/rm rm
+	substituteInPlace ./undo.c \
+		--replace /bin/cat cat
+  '';
+
+  buildPhase = ''
+    make -j6 V=1 p
+    make image
+  '';
+
+  installPhase = ''
+    ./utils/bk-* $out/bitkeeper
+    mkdir -p $out/bin
+    $out/bitkeeper/bk links $out/bin
+    chmod g-w $out
+  '';
+
+  meta = {
+    description = "BitKeeper";
```

Comment:
Please use a description like "A distributed version control system" or something to that effect.